spring-boot-security-taglibs/src/main/java/org/baeldung/security/HomeController.java

```diff
@@ -0,0 +1,16 @@
+package org.baeldung.security;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@Controller
+@RequestMapping("/")
+public class HomeController {
+    @RequestMapping("")
```

Comment:
Would @GetMapping be more appropriate here? Also, I'm unclear whether or not "" is actually necessary here. Could you remove the annotation altogether?